### PR TITLE
Update ContribSummitInfo

### PR DESCRIPTION
**Pull request by parispittman**
added CIVS info/call for topics and fixed some nits

### CODE REVIEW

File: community/2017-events/12-dev-summit/ContribSummitInformation.md [hunk 1]
```diff
@@ -27,20 +27,20 @@ Austin, TX 78701
 Selected attendees are Kubernetes upstream contributors including SIG members and lottery winners from a [members of standing](https://github.com/kubernetes/community/blob/master/community-membership.md) pool.
 We realize that this is not the best approach now that our community is growing in exponential ways. In 2018, we will be unfolding a new strategy for bringing our contributors together so all voices can be heard. 
 
-Invitees will begin going out on October 19th.
+Invitees went out on October 19th.
```
Comment (cblecker) [inline community/2017-events/12-dev-summit/ContribSummitInformation.md]: nit: Invites vs Invitees

File: community/2017-events/12-dev-summit/ContribSummitInformation.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ In some sense, the summit is a real-life extension of the community meetings and
 * All day event with happy hour reception to close  
 
 **Agenda:**  
-Times in CST 
+Times in CST
```
Comment (cblecker) [inline community/2017-events/12-dev-summit/ContribSummitInformation.md]: nit: added whitespace?

File: community/2017-events/12-dev-summit/ContribSummitInformation.md [hunk 3]
```diff
@@ -27,20 +27,20 @@ Austin, TX 78701
...
 "loosely-structured unconference"
 Rather than speakers and presentations, we will have moderators/facilitators and discussion topics, alongside all-day completely unstructured hacking.
 
 The discussion sessions will be in an open fishbowl format — rings of chairs, with inner rings driving the discussion — where anyone can contribute. The various sessions will be proposed and voted on by the community in the weeks leading up to the event. This allows the community to motivate the events of the day without dedicating precious day-of time to choosing the sessions and making a schedule.
 
-There will be 3 rooms dedicated to these sessions running in parallel all day. Each session should last between 45 minutes and an hour.
+There will be 2 rooms dedicated to these sessions running in parallel all day. Each session should last between 45 minutes and an hour.
 
-Then, there will be 2 smaller rooms for hacking / unstructured discussion all day. 
+Then, there will be a smaller room for hacking / unstructured discussion all day.
```
Comment (cblecker) [inline community/2017-events/12-dev-summit/ContribSummitInformation.md]: nit: There will {also|additionally} be

**Comment by castrojo** [comment]
/lgtm

This doc is fast revving for an event so we're going to just merge quickly.

**Pull request merged**